avaframe/ana5Hybrid/hybridTools.py

```diff
@@ -0,0 +1,416 @@
+import pathlib
+import numpy as np
+import shapefile
+import matplotlib.pyplot as plt
+
+# Local imports
+import avaframe.com1DFA.DFAtools as DFAtls
+from avaframe.in3Utils import geoTrans
+import avaframe.out3Plot.plotUtils as pU
+import avaframe.out3Plot.outCom1DFA as outCom1DFA
+
+debug = False
+
+
+def getCom1DFAPath(particlesList, dem):
+    """ compute part, mass and energy averaged path from particles
+
+    Also returns the averaged velocity ant kinetik energy associated
+
+    Parameters
+    -----------
+    particlesList: list
```

Comment:
Just to make sure: all these functions rely on the profile being from top to bottom. Can we rely on the particleList being in the correct order?

Reply:
The list of particles dictionary are assumed to be ordered for increasing times.